tests/checkbox/automated/test-01-navigate-to-unchecked-checkbox-reading.nvda.json

```diff
@@ -0,0 +1,244 @@
+[
+  {
+    "nav": [
+      "reference/two-state-checkbox.html"
+    ]
+  },
+  {
+    "press_until_contains": [
+      "h",
+      "Sandwich Condiments"
+    ]
+  },
+  {
+    "clear_output": []
+  },
+  {
+    "press": [
+      "x"
+    ]
+  },
+  {
+    "assert_role": [
+      "checkbox"
```

Comment:
If there is going to be a test file for each AT, do is maybe make more sense for the normalization of role from `checkbox` to `check box` to live here? (I'm not sure...)

Reply:
I think it's better to have the mapping in the AT's repo (if open source, at least), so that the test can be applicable across versions of the AT even if they changed how a role is conveyed. If it's in `aria-at` then it makes things annoying both for the AT vendor and the aria-at CG when it changes (though may be rare, but still).